binrz/rz-test/rz-test.c

```diff
@@ -1089,6 +1205,26 @@ static void interact_break(RzTestResultInfo *result, RzPVector *fixup_results) {
 	replace_cmd_kv_file(result->test->path, line_begin, line_end, "BROKEN", "1", fixup_results);
 }
 
+static void interact_break_asm(RzTestResultInfo *result) {
+	assert(result->test->type == RZ_TEST_TYPE_ASM);
+	RzAsmTest *test = result->test->asm_test;
+	replace_asm_test(result->test->path, test->line,
```

Comment:
what about passing `RzAsmTest` instead?

Reply:
Can't, because the other caller of `replace_asm_test` needs to pass different args and the remaining intersection that could be merged by this is only the single arg `test->line`.